docs/user_guide/getting_started.md

```diff
@@ -88,14 +88,14 @@ cd recipes
 cp template.env .env
 cp local_nb_nodes.template.json local_nb_nodes.json
 ```
-3. Change `NB_API_QUERY_URL` in the `.env` file
+3. Change the default credentials for your graph store (and optionally set any additional security variables as needed) following [these instructions](config.md#change-security-relevant-variables).
```

Comment:
If this is now a required, separate step, why do we have default credentials in the recipes at all? I understand the need to tell folks what they can do. But why not put a couple of links in a section on "optional" next steps? I know the "password change needs volume wipe" limitation. But the point of getting started is to go from "what is this thing" to "oh look it runs, I'm smart" in as little time and effort as possible